neuralmonkey/decoders/transformer.py

```diff
@@ -280,8 +319,11 @@ def body(*args) -> LoopState:
                 # (batch, state_size)
                 output_state = last_layer.temporal_states[:, -1, :]
 
-                logits = tf.matmul(output_state, self.decoding_w)
-                logits += self.decoding_b
+                # See train_logits definition
+                logits_unbiased = tf.matmul(
+                    output_state,
+                    self.decoding_w)
+                logits = logits_unbiased + self.decoding_b
```

Comment:
proč je tu tahle změna? logits_unbiased se nikde dál nepoužívaj

Reply:
Bud trochu konzistentni. Presne takto jsi to tam mel ty, i presto, ze jsi potom logits_unbiased nikde dal nepouzival. Ale klidne to prejmenuju.

Reply:
no koukám že se ze dvou čitelnejch řádků stalo pět nečitelnejch a dělaj to samý